src/mono/netcore/sample/wasm/browser/Wasm.Browser.Sample.csproj

```diff
@@ -40,9 +41,9 @@
     <PropertyGroup>
         <WasmAppDir>$(AppDir)</WasmAppDir>
         <WasmAOTDir>$(MSBuildProjectDirectory)\$(PublishDir)\</WasmAOTDir>
-        <WasmMainAssemblyPath>$(MSBuildProjectDirectory)\bin\WasmSample.dll</WasmMainAssemblyPath>
+        <WasmMainAssemblyPath>$(MSBuildProjectDirectory)\bin\$(SampleName).dll</WasmMainAssemblyPath>
```

Comment:
Would it be more direct to just use `$(AssemblyName)` for all occurences of `$(SampleName)`

Reply:
yeah, thanks, perhaps it makes sense to avoid this intermediate property.